objectPerformance/src/turnon_collection.py

```diff
@@ -142,8 +142,8 @@ def _match_test_to_ref(self):
             pt_max = ak.argmax(ref_test["test"]["pt"][pass_dR], axis=-1,
                                keepdims=True)
             if ("iso" not in suffix):
-                self.numerators["ref"][test_obj] = ref_test["ref"][suffix][pass_dR][pt_max][:, :, 0]  # noqa
-            self.numerators["test"][test_obj] = ref_test["test"][suffix][pass_dR][pt_max][:, :, 0]  # noqa
+                self.numerators["ref"][test_obj] = ref_test["ref"][suffix][pass_dR][pt_max][:, 0]  # noqa
+            self.numerators["test"][test_obj] = ref_test["test"][suffix][pass_dR][pt_max][:, 0]  # noqa
```

Comment:
Have you tested this? Does the modification alter the plots? Why did we have three levels of nestedness before?

Reply:
Yes, I have tested it on a few objectPerformance configs and the plots seem to be reproduced this way. But tbh I have not understood from first principles what exactly the change here is.

Reply:
could it be due to the awkward version change?
btw, there is a dedicated function to the get the first element in the event: [`ak.firsts`](https://awkward-array.org/doc/main/reference/generated/ak.firsts.html) which also fills the empty rows with `None`s. Should we switch to that instead?

Reply:
I tested `ak.firsts` and indeed it produces the same result judging from the electron iso plots [here](https://dhundhausen.web.cern.ch/L1_PhaseII/ak_firsts/) with `ak.firsts` and in the directory above that with the slicing.